opentech/apply/determinations/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import DeterminationCreateOrUpdateView, DeterminationDetailView
+
+app_name = 'determinations'
+
+urlpatterns = [
+    path('determination/', DeterminationCreateOrUpdateView.as_view(), name="form"),
+    path('determination/<int:pk>', DeterminationDetailView.as_view(), name="detail"),
```

Comment:
Determination is a one to one with the submission. So should the urls be `/determination/create` and `/determination`

Reply:
👍